modular_skyrat/modules/modular_items/lewd_items/code/lewd_helpers/species.dm

```diff
@@ -0,0 +1,74 @@
+/datum/species/proc/throw_arousalalert(level, atom/movable/screen/alert/aroused_x/arousal_alert, mob/living/carbon/human/targeted_human)
+	targeted_human.throw_alert("aroused", /atom/movable/screen/alert/aroused_x)
+	arousal_alert?.icon_state = level
+	arousal_alert?.update_icon()
+
+/datum/species/proc/overlay_pain(level, atom/movable/screen/alert/aroused_x/arousal_alert)
+	arousal_alert?.cut_overlay(arousal_alert.pain_overlay)
+	arousal_alert?.pain_level = level
+	arousal_alert?.pain_overlay = arousal_alert.update_pain()
+	arousal_alert?.add_overlay(arousal_alert.pain_overlay)
+	arousal_alert?.update_overlays()
+
+/datum/species/proc/overlay_pleasure(level, atom/movable/screen/alert/aroused_x/arousal_alert)
+	arousal_alert?.cut_overlay(arousal_alert.pleasure_overlay)
+	arousal_alert?.pleasure_level = level
+	arousal_alert?.pleasure_overlay = arousal_alert.update_pleasure()
+	arousal_alert?.add_overlay(arousal_alert.pleasure_overlay)
+	arousal_alert?.update_overlays()
+
+/datum/species/proc/handle_arousal(mob/living/carbon/human/target_human, atom/movable/screen/alert/aroused_x)
+	var/atom/movable/screen/alert/aroused_x/arousal_alert = target_human.alerts["aroused"]
+	if(target_human.client?.prefs?.read_preference(/datum/preference/toggle/erp/sex_toy))
+		switch(target_human.arousal)
+			if(-INFINITY to AROUSAL_MINIMUM_DETECTABLE)
+				target_human.clear_alert("aroused", /atom/movable/screen/alert/aroused_x)
+				if(target_human.arousal < AROUSAL_MINIMUM)
+					target_human.arousal = AROUSAL_MINIMUM // To prevent massively negative values that break the lewd system for some.
+			if(AROUSAL_MINIMUM_DETECTABLE to AROUSAL_LOW)
+				throw_arousalalert("arousal_small", arousal_alert, target_human)
```

Comment:
Those strings should be defines, and should be stored in a variable to then be used for the proc call once the switch statement is over.